src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -759,5 +785,75 @@ private static bool IsPotentialFilterCharacter(char c)
                 || char.IsNumber(c)
                 || c == '_';
         }
+
+        private static int GetAggressiveDefaultsMatch(List<MatchResult<VSCompletionItem>> itemsWithMatch, ImmutableArray<string> defaults)
+        {
+            Debug.Assert(!defaults.IsDefaultOrEmpty);
+
+            foreach (var defaultText in defaults)
+            {
+                for (var i = 0; (i < itemsWithMatch.Count); ++i)
+                {
+                    var itemWithMatch = itemsWithMatch[i];
+                    if (itemWithMatch.RoslynCompletionItem.FilterText == defaultText)
```

Comment:
why are we looking at filter-text?  we should likely be looking at display text right (in other words, we're seeing that two things in the *display* match up, what is in teh buffer, and waht is shown the user.  if these are the same, we should pick ours?

Reply:
Makes sense, I changed it to DisplayText. FYI @dpugh